gen_js_api.opam

```diff
@@ -9,10 +9,18 @@ authors: [
 homepage: "https://github.com/LexiFi/gen_js_api"
 bug-reports: "https://github.com/LexiFi/gen_js_api/issues"
 license: "MIT"
-dev-repo: "https://github.com/LexiFi/gen_js_api.git"
+dev-repo: "git+https://github.com/LexiFi/gen_js_api.git"
 build: [["dune" "build" "-p" name "-j" jobs]]
 depends: [
+  "ocaml" {>= "4.06.0" & < "4.09.0"}
```

Comment:
Why is the `< 4.09.0` contraint needed ?

Reply:
I migth have taken this from the opam-repository. Might not be needed anymore

Reply:
Also, don't we need `ocaml-migrate-parsetree` as well?

Reply:
Indeed. I won't be able to do any change in the next few days. Feel free to push to the branch directly if you want to move the change forward.

Reply:
fixed